src/redux/selectors.js

```diff
@@ -0,0 +1,11 @@
+export function getCurrentCity(state) {
```

Comment:
good job with selectors :)
You get the jist of it and when you tackle 57hours, you'll see how complex (and sometimes ugly) they can be :)